docker/Emulator_x86

```diff
@@ -7,6 +7,18 @@ LABEL maintainer "Budi Utomo <budtmo.os@gmail.com>"
 #=============
 WORKDIR /root
 
+#===========
+# Polyverse
```

Comment:
This commit can reduce the image size by 1GB.
You can take a look at [my docker repo](https://hub.docker.com/r/trinhpham/docker-android-x86-9.0/tags)
![image](https://user-images.githubusercontent.com/9128061/60330982-d0a73b00-99bd-11e9-8b87-9ba9aa8e3a26.png)